cmd/configure_raid.go

```diff
@@ -48,7 +48,12 @@ var configureRaidCmd = &cobra.Command{
 				Level:   GetString(cmd, "raid-level"),
 			}
 
-			if err = raidArray.Create(ctx, GetString(cmd, "raid-type")); err != nil {
+			raidType := GetString(cmd, "raid-type")
+			if raidType == "" {
+				raidType = "linuxsw"
```

Comment:
Think that might be something to stick in the bmc-toolbox/common package?    Starting to feel a lot less bmc specific by now..

Reply:
Sounds about right - we can have that in the common package. 

The common package was created under the bmc-toolbox repository since bmclib depends on it and because the bmc-toolbox org ownership was more 'open' compared to this metal-toolbox org. Going ahead if this org is handed over to a neutral entity or has external maintainers, admins then I'd be happy to move it here.